### PR TITLE
Insert payment patch

### PR DESCRIPTION
**Pull request by JoeDicker**
- Changed typo in PaymentsController "usermame" to "username" doPost()

- Changed "id" to "ID" in MembersRepo get() override

- Changed the insert() override in PaymentsRepo to get columns

- Changed "payments" to "PAYMENTS" in PaymentsRepo insert() override

*Currently does not update user balance but **_should_** work once Feature/show annual turnover and provisional members PR has been approved

### CODE REVIEW

File: XYZDrivers/src/main/java/com/xyzdrivers/repositories/PaymentsRepo.java [hunk 1]
```diff
@@ -101,7 +101,8 @@ public void delete(MembershipPayment model) {
     @Override
     public void insert(MembershipPayment model) throws RepositoryException {
         try {
-            sqlService.insert("payments", new Object[]{
+            String columns = ("MEM_ID, TYPE_OF_PAYMENT, AMOUNT, DATE, TIME");
+            sqlService.insert("PAYMENTS", new Object[]{
```
Comment (lenfred123) [inline XYZDrivers/src/main/java/com/xyzdrivers/repositories/PaymentsRepo.java]: Columns still aren't being passed to the insert here.

Comment (JoeDicker) [reply XYZDrivers/src/main/java/com/xyzdrivers/repositories/PaymentsRepo.java]: Oops!

**Review by lenfred123 — APPROVED** [review]
Cool, merge away.

File: XYZDrivers/src/main/java/com/xyzdrivers/repositories/PaymentsRepo.java (hunk 1 above)
Comment (JoeDicker) [reply XYZDrivers/src/main/java/com/xyzdrivers/repositories/PaymentsRepo.java]: Sorted.

**Pull request merged**